config/locales/en/main.yml

```diff
@@ -144,7 +144,7 @@ en:
   errors:
     # our messages are all complete sentences so we use the colon
     format: "%{attribute}: %{message}"
-
+    file_upload: "There was an error uploading the file."
```

Comment:
I moved the error out of active record bc it will be used in generic file uploading. Is there a better place to put it?

Reply:
@smoyth how would I figure out if it's a validation error? I see "validates_attachment_presence" in /models/media/object.rb but I dont' know how to see what the text of that error would be?

Reply:
If it were being used that way, the symbol `file_upload` would need to be passed to `error.add`. So if that's not showing up anywhere I think you're golden.

Reply:
the old one was "generic" but that's not getting added to errors anywhere either so I think it's good to go.